webapp/channels/src/components/admin_console/schema_admin_settings.scss

```
@@ -29,3 +29,8 @@
 .config-section:nth-child(1) {
     margin-top: 0;
 }
+
+.admin-header-beta-badge {
+    margin-right: auto !important;
+    margin-inline: 8px;
```

you have margin-inline here which will add 8px at the start and the end, but we have the margin-right value set with an important flag. 
```suggestion
    margin-right: auto;
    margin-inline-start: 8px;
```